layouts/default.vue

```diff
@@ -32,8 +32,9 @@
 </template>
 
 <script lang="ts">
-import Vue from 'vue'
-import { LinkPropertyHref, MetaInfo } from 'vue-meta'
+import { NuxtConfig } from '@nuxt/types'
```

Comment:
型のみのインポートなので、import type文にできそうです。

```suggestion
import type { NuxtConfig } from '@nuxt/types'
```

Reply:
ありがとうございます！
使わせていただきました